### PR TITLE
Created models & relationships

### PR DESCRIPTION
**Pull request by victor-sopot**

### CODE REVIEW

File: app/Department.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+use Illuminate\Database\Eloquent\Model;
+
+class Department extends Model
+{
+	public function users() {
+		return $this->hasMany('App\User');
+	}
+
+	public function organisation() {
```
Comment (Tanprasit) [inline app/Department.php]: Change to 'organisations' because an HR department can exist in Oracle and HP for example.

**Review by Tanprasit — CHANGES_REQUESTED** [review]
Fix the recommendation

File: app/Department.php (hunk 1 above)
Comment (victor-sopot) [reply app/Department.php]: So organisations and departments need to be many to many. (pivot table)

Comment (Tanprasit) [reply app/Department.php]: Yep @djdavies agree?

**Review by djdavies — COMMENTED** [review]
Yes, seems like we need a pivot for that. It would also mean when an organisation registers with us, they will not need to manually enter so much information if it's already pre-existing (we'll probably generate a load).

I wouldn't change it to a pluralisation, as the relationship itself is indeed wrong as it's a many-to-many.

**Comment by victor-sopot** [comment]
Ready to be reviewed

**Review by Tanprasit — APPROVED** [review]

**Pull request merged**